### DISCUSSION TITLE
Filament Community

### DISCUSSION THREAD

**Discussion by vanvanni** [filamentphp/filament#490] [Ideas]
Hi,

I was wondering if it was possible to create a dedicated Filament community instead of having one channel within the Laravel Livewire server. This is so we can have dedicated channels to certain topics and maybe even have a dedicated Plugins channel(see idea below) to share helpful sets of filament code.

**Plugins:**
I wonder if it would be possible to make plugins or add-ons for Filament so we can share certain utilities with each other. I didn't dive in deep but it looks like with a CustomPage you could maybe make a log viewer that is useful to a lot of filamentadmin users. If we have a central space to share them this could help the community and adoption a lot. What are your thoughts on this?

**Comment by ryangjchandler** [comment]
@xJoeyv Plugins are definitely possible. We provide a `PluginServiceProvider` that you can extend.

**Reply by vanvanni** [reply]
> @xJoeyv Plugins are definitely possible. We provide a `PluginServiceProvider` that you can extend.

I was now about write I saw that plugins are possible but you already responded. Would like to see if we can have a "marketplace" or "hub" for this. But as mentioned before a Discord server would be nice aswell. Love to works and feels of Filament.

**Reply by ryangjchandler** [reply]
We've spoken about a marketplace too before.

**Reply by vanvanni** [reply]
> We've spoken about a marketplace too before.

And... 😊

**Reply by vanvanni** [reply]
> We've spoken about a marketplace too before.

And what about a dedicated Discord? I could help set it up or to manage it.

**Reply by danharrin** [reply]
Marketplace is coming with v2 ;)

As for a dedicated Discord, if there is enough support for it then i'll do the same :)

**Reply by danharrin** [reply]
Marketplace is being launched today! https://filamentphp.com/plugins
